src/main/java/com/fitwsarah/fitwsarah/appointmentsubdomain/datalayer/Appointment.java

```diff
@@ -50,4 +48,32 @@ public Appointment(String availabilityId, String userId, String serviceId, Statu
         this.date = date;
         this.time = time;
     }
+
+    public void setStatus(Status status) {
+        this.status = status;
+    }
+
+    public void setLocation(String location) {
+        this.location = location;
+    }
+
+    public void setFirstName(String firstName) {
+        this.firstName = firstName;
+    }
+
+    public void setLastName(String lastName) {
+        this.lastName = lastName;
+    }
+
+    public void setPhoneNum(String phoneNum) {
+        this.phoneNum = phoneNum;
+    }
+
+    public void setDate(String date) {
+        this.date = date;
+    }
+
+    public void setTime(String time) {
+        this.time = time;
+    }
```

Comment:
Can you explain why you added getter and setter for the appointments?

Reply:
Inside of my service impl method I am using these setters to set the new value when updating.